packages/spycloud/data_stream/breach_record/_dev/test/pipeline/test-breach-record.log-expected.json

```
@@ -0,0 +1,204 @@
+                    ],
+                    "cc": {
+                        "bin": "489486",
+                        "expiration": "06/2025",
+                        "last_four": "1237",
+                        "number": "3fdd0ce028ffaa147afdb6461f6ce95f8c07f484"
+                    },
+                    "company_name": "ABC Corporation",
+                    "country": {
+                        "code": "US",
+                        "name": "United States"
+                    },
+                    "display_resolution": "1920x1080",
+                    "document_id": "3350f1da-fa39-4415-b2cc-02057e2fbe99",
+                    "domain": "example1.com",
+                    "email": {
+                        "domain": "example.com",
+                        "username": "john.doe",
+                        "value": "john.doe@example.com"
+                    },
+                    "first_name": "John",
+                    "full_name": "John Doe",
+                    "homepage": "https://www.chess.com/member/sarahjoh",
+                    "industry": "Technology",
+                    "infected": {
+                        "machine_id": "ABC123",
+                        "path": "/documents/confidential",
+                        "time": "2023-01-15T12:30:45.000Z"
+                    },
+                    "ip_addresses": [
+                        "89.160.20.128",
+                        "89.160.20.112"
+                    ],
+                    "job_title": "Software Engineer",
+                    "keyboard_languages": [
+                        "English",
+                        "Spanish"
+                    ],
+                    "last_name": "Doe",
+                    "log_id": "76afa48107ec32f51a2aba4a314357c1e69d2267f1b04bf1afc948d0f77b1658",
+                    "password": {
+                        "plaintext": "******",
+                        "type": "alphanumeric",
+                        "value": "P@ssw0rd123"
```

Most users don't like passwords being displayed. You can create a user-configurable field such as `hide_sensitive_data` (default: true) and if it is set to `true`, mask the password(s) and other sensitive data in ingest pipeline.